main.go

```diff
@@ -34,6 +34,11 @@ func mainErr() error {
 	app.Before = func(ctx *cli.Context) error {
 		if ctx.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
+			logrus.Debugf("Loglevel set to [%v]", logrus.DebugLevel)
+		}
+		if ctx.GlobalBool("trace") {
```

Comment:
not sure if you care but if you do --trace --debug --quiet you'll get two outputs for setting the log level and then it'll be quiet. maybe check quiet first and do an else for the other two?